frontend/src/views/Transactions.vue

```diff
@@ -147,6 +168,15 @@ export default defineComponent({
           translationPrefix : 'transactions.table',
         },
         filters: {
+          accounts: {
+            align   : 'left',
+            value   : null,
+            options : [],
+            handler : ({row, value} : { row : Transaction, value : Account }) : boolean => {
```

Comment:
Why do you input an object here instead of two parameters. Would simplify the type annotations. Same as with the search handler

Reply:
The idea was that not all handlers need `value` or `row` and that would let them choose.

Reply:
But, good point, i've changed it to `(value,row)`

I also refactor the other search/filters handlers to all accept `(value , row)`  some had `(row, value)` before